using-timescaledb/continuous-aggregates.md

```diff
@@ -248,6 +244,111 @@ hypertable with a continuous aggregate.
 
 ---
 
+### Advanced Topics [](advanced-usage)
+
+#### Continuous Aggregates using Integer-Based Time [](create-integer)
+
+Usually, continuous aggregates are defined on a [date/time-type](https://www.postgresql.org/docs/current/datatype-datetime.html) column, but it is
+also possible to create your own custom scheme for handling
+aggregation for tables that are using an integer time
+column. This can be useful if you have tables that use other measures
+of time that can be represented as integer values, such as nanosecond
+epochs, minutes since founding date, or whatever is suitable for your
+application.
+
+As an example, suppose that you have a table with CPU and disk usage
+for some devices where time is measured in
+[microfortnights][fff-system]. Since you are using an integer-valued
+column as time, you need to provide the chunk time interval when
+creating the hypertable. In this case, let each chunk consist of a
+millifortnight (which is about 20 minutes).
+
+```sql
+CREATE TABLE devices(
+  time BIGINT,        -- Timestamp in microfortnights
+  cpu_usage INTEGER,  -- Total CPU usage
+  disk_usage INTEGER, -- Total disk usage
+  PRIMARY KEY (time)
+);
+
+SELECT create_hypertable('devices', 'time',
+                         chunk_time_interval => 1000);
+```
+
+To define a continuous aggregate on a hypertable that is using an integer
+dimension, it is necessary to have a function to get the current
+timestamp in whatever representation that you are using and set it for
```

Comment:
It might be just personal opinion:
```suggestion
dimension, it is necessary to have a function to get the current
time in whatever representation that you are using and set it for
```

Reply:
I'm talking about timestamp everywhere else and am using the `CURRENT_TIMESTAMP` to get the epoch, so I think that "timestamp" is more consistent.

Reply:
Section `Using Continuous Aggregates` in this file uses `current time`, so `current timestamp` is not consistent with the rest of the document. I don't see that you use `timestamp` in the text, except here. `current time` is easier to follow and will be consistent with the rest of the document.

Reply:
Good point. Switched to use `current_time` and removing uses of "timestamp" in this commit. There are still some mention of timestamps in the chapter, but that is for a separate PR.